tests/models/Entry_test.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+//TEST SETUP
+let Entry;
+let assert = chai.assert;
+let GetEntry = () =>  {
+  return {
+    id: 1,
+    title: 'Test Entry',
+    guid: '1',
+    link: 'Test Link 1',
+    updated: new Date(),
+    content: 'Test Content 1',
+    content_hash: 'Test Content Hash 1',
+    cached_content: 'Test Cached Content 1',
+    date_entered: new Date(),
+    date_updated: new Date(),
+    lang: 'English'
+  };
+};
+
+describe('Entry', function() {
+  co.wrap(function* () {
+    let sequelize = yield getTestDB();
+    Entry = sequelize.models.Entry;
+  })();
+  it('Saves and retrieves successfully', co.wrap(function* () {
+    yield Entry.sync({force: true});
+    let insert = GetEntry();
+    yield Entry.create(insert);
+
+    let retrieve = yield Entry.findById('1');
+
+    expect(retrieve.id).to.equal(insert.id);
+    expect(retrieve.title).to.equal(insert.title);
+    expect(retrieve.guid).to.equal(insert.guid);
+    expect(retrieve.link).to.equal(insert.link);
+    expect(retrieve.updated.getTime()).to.equal(insert.updated.getTime());
+    expect(retrieve.content).to.equal(insert.content);
+    expect(retrieve.content_hash).to.equal(insert.content_hash);
+    expect(retrieve.cached_content).to.equal(insert.cached_content);
+    expect(retrieve.date_entered.getTime()).to.equal(insert.date_entered.getTime());
+    expect(retrieve.date_updated.getTime()).to.equal(insert.date_updated.getTime());
+    expect(retrieve.lang).to.equal(insert.lang);
+  }));
+  it('Auto increments on id', co.wrap(function* () {
+    yield Entry.sync({force: true});
+
+    let insert1 = GetEntry();
+    let insert2 = GetEntry();
+    delete(insert1.id);
+    delete(insert2.id);
+    yield Entry.create(insert1);
+    yield Entry.create(insert2);
+
+    let retrieve = yield Entry.findAll();
+    expect(retrieve).to.have.lengthOf(2);
+    expect(retrieve[1].id).to.equal(retrieve[0].id + 1);
+  }));
+  it('Requires id to be an integer', co.wrap(function* () {
+    let sequelize = yield getTestDB();
+    let Entry = sequelize.models.Entry;
+    yield Entry.sync({force: true});
+
+    let insert = GetEntry();
+    insert.id = 'STRING';
+
+    let success = false;
+    yield Entry.create(insert).then(function(response) {
```

Comment:
From the author of the `co` library: https://medium.com/@tjholowaychuk/callbacks-vs-coroutines-174f1fe66127#3f51

No worries if you can't get it working though. It's concise either way, just figured I'd mention it as its a nice thing about async generators.